api/app/core/database.py

```
@@ -0,0 +1,106 @@
+    - Commits on success
+    - Rolls back on error
+    - Always closes the session
+    """
+    async with async_session_maker() as session:
+        try:
+            yield session
+            # Commit transaction on success
+            # This commits all changes made during the request
+            # In serverless, this must complete before the function terminates
+            await session.commit()
+        except Exception as e:
```

Ruff (0.14.10)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)